editor/editor_inspector.cpp

```diff
@@ -321,9 +322,9 @@ void EditorProperty::_notification(int p_what) {
 					color2.b *= 1.2;
 				}
 				if (rtl) {
-					draw_texture(reload_icon, Vector2(size.width - revert_rect.position.x - reload_icon->get_width(), revert_rect.position.y), color2);
+					draw_texture(reload_icon, Vector2(size.width - revert_rect.position.x - (padding.width / 2) - reload_icon->get_width(), revert_rect.position.y + (padding.height / 2)), color2);
 				} else {
-					draw_texture(reload_icon, revert_rect.position, color2);
+					draw_texture(reload_icon, revert_rect.position + Vector2((padding.width / 2), padding.height / 2), color2);
```

Comment:
```suggestion
					draw_texture(reload_icon, revert_rect.position + padding / 2, color2);
```
No need to decompose and add back

Reply:
Maybe I don't understand you correctly. But without this the icon ends up at the top clickable area.

With + Vector2
![Screenshot_20240727_111644](https://github.com/user-attachments/assets/f5d3ae3d-4cc8-48ba-b239-bcefb48e2fcb)

Without:
![Screenshot_20240727_111932](https://github.com/user-attachments/assets/824831bd-0e43-474f-9d9d-47532eda583d)

Reply:
It really shouldn't as what I suggested is identical, did you copy it correctly?

Reply:
Okay, got it now. Thanks!